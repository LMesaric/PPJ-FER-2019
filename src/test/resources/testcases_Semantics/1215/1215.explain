semantic: can't initialize scalar value with array
note: 'alo = "bok"' on line 21
