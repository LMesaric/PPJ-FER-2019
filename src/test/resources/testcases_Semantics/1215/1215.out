<init_deklarator> ::= <izravni_deklarator> OP_PRIDRUZI(21,=) <inicijalizator>
